scripts/generate_proto_stubs.sh

```diff
@@ -66,3 +66,5 @@ PROTO_FILES=$(grep "generate_proto.*google" $PYTHON_PROTOBUF_DIR/python/setup.py
 
 # And regenerate!
 protoc_install/bin/protoc --proto_path=$PYTHON_PROTOBUF_DIR/src --mypy_out=$REPO_ROOT/stubs/protobuf $PROTO_FILES
+isort $REPO_ROOT/stubs/protobuf
+black $REPO_ROOT/stubs/protobuf
```

Comment:
We are still in the venv activated previously. I don't think it has black and isort?

Also, do we have any way to test this script, besides cherry-picking every manual change?

Reply:
Good catch. I guess it worked during my tests, since I still had my "proper" venv activated.

Reply:
I didn't know that you can nest virtualenvs and it'll work, but now that I think about it, it makes sense, since the activate script mostly just adds things to `$PATH`.

Reply:
I didn't know that either, never tried it before, but that's the only explanation I have.